turbo.json

```diff
@@ -61,10 +61,7 @@
     },
     "dev": {
       "persistent": true,
-      "cache": false,
-      "dependsOn": [
-        "^build"
```

Comment:
why?

Reply:
why do we need it?

Reply:
Otherwise dev might not reflect changes in other packages